File: Core/Entities/Customer.cs
using System.Collections.Generic;
using System;

namespace Core.Entities
{
    public class Customer : BaseEntity
    {
        public ICollection<Invoice> Invoices { get; set; }
        public string Email { get; set; }
        public string Appliance { get; set; }
        public string Brand { get; set; }
        public string RequestedService { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string PostalCode { get; set; }
        public string FullName { get; set; }
        public string PhoneNumber { get; set; }
        public string ModelNumber { get; set; }
        public DateTimeOffset RegisteredDate { get; set; } = DateTimeOffset.Now;

    }
}

File: client/src/app/shared/models/customer.ts
export interface ICustomer {
    id: number;
    email: string;
    appliance: string;
    brand: string;
    requestedService: string;
    address: string;
    city: string;
    postalCode: string;
    fullName: string;
    phoneNumber: string;
    modelNumber: string;
    registeredDate: string;
    invoices: string[];
}